feature/foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -175,10 +175,7 @@ fun ForYouScreen(
             }
         }
 
-        val tag = "forYou:feed"
-
-        val lazyGridState = rememberLazyGridState()
-        TrackScrollJank(scrollableState = lazyGridState, stateName = tag)
+        TrackScrollJank(scrollableState = rememberLazyGridState(), stateName = "forYou:feed")
```

Comment:
I'm not too sure how tracking Jank works, but is it an error to pass the same instance of `LazyGridState` to `TrackScrollJank` and `LazyVerticalGrid`? It seems the stats tracked by `TrackScrollJank` won't match the scroll behavior of the Composable.